cms/envs/aws.py

```diff
@@ -357,3 +357,20 @@
 
 PROCTORING_BACKEND_PROVIDER = AUTH_TOKENS.get("PROCTORING_BACKEND_PROVIDER", PROCTORING_BACKEND_PROVIDER)
 PROCTORING_SETTINGS = ENV_TOKENS.get("PROCTORING_SETTINGS", PROCTORING_SETTINGS)
+
+##### Third-party auth options ################################################
+if FEATURES.get('ENABLE_THIRD_PARTY_AUTH'):
+    AUTHENTICATION_BACKENDS = (
+        ENV_TOKENS.get('THIRD_PARTY_AUTH_BACKENDS', [
```

Comment:
I see now. I think storing this as either configuration models or in configuration could keep it in sync with the LMS settings. The problem with copy-pasting this is that the two lists could grow out of sync. @bradenmacdonald - looks like you've worked quite a bit on third party auth - any suggestions?